notes.md

```diff
@@ -104,23 +104,23 @@ file name   | `SegmentUID`                       | `PrevUID`
+Soft Linking is used by codec chapters. They can reference another `Segment` and jump to that `Segment`. The way the `Segments` are described are internal to the chapter codec and unknown to the Matroska level. But there are `Elements` within the `Info Element` (such as `ChapterTranslate`) that can translate a value representing a `Segment` in the chapter codec and to the current `SegmentUID`. All `Segments` that could be used in a `Linked Segment` in this way SHOULD be marked as members of the same family via the `SegmentFamily Element`, so that the `Matroska Player` can quickly switch from one to the other.
 
 # Track Flags
 
 ## Default flag
 
-The "default track" flag is a hint for the playback application and SHOULD always be changeable by the user. If the user wants to see or hear a track of a certain kind (audio, video, subtitles) and hasn't chosen a specific track, the player SHOULD use the first track of that kind whose "default track" flag is set to "1". If no such track is found then the first track of this kind SHOULD be chosen.
+The "default track" flag is a hint for a `Matroska Player` and SHOULD always be changeable by the user. If the user wants to see or hear a track of a certain kind (audio, video, subtitles) and hasn't chosen a specific track, the `Matroska Player` SHOULD use the first track of that kind whose "default track" flag is set to "1". If no such track is found then the first track of this kind SHOULD be chosen.
 
 Only one track of a kind MAY have its "default track" flag set in a segment. If a track entry does not contain the "default track" flag element then its default value "1" is to be used.
 
 ## Forced flag
 
-The "forced" flag tells the playback application that it MUST display/play this track or another track of the same kind that also has its "forced" flag set. When there are multiple "forced" tracks, the player SHOULD determined based upon the language of the forced flag or use the default flag if no track matches the use languages. Another track of the same kind without the "forced" flag may be use simultaneously with the "forced" track (like DVD subtitles for example).
+The "forced" flag tells the `Matroska Player` that it MUST display/play this track or another track of the same kind that also has its "forced" flag set. When there are multiple "forced" tracks, the `Matroska Player` SHOULD determined based upon the language of the forced flag or use the default flag if no track matches the use languages. Another track of the same kind without the "forced" flag may be use simultaneously with the "forced" track (like DVD subtitles for example).
```

Comment:
s/determined/determine/ but all that proof reading can go in a separate patch.

Reply:
I'm right here so I'll Girl Scout and fix it now 💪